selfdrive/car/honda/carcontroller.py

```diff
@@ -148,11 +146,22 @@ def update(self, enabled, CS, frame, actuators,
     can_sends.append(hondacan.create_steering_control(self.packer, apply_steer,
       lkas_active, CS.CP.carFingerprint, idx, CS.CP.openpilotLongitudinalControl))
 
-    # Send dashboard UI commands.
-    if (frame % 10) == 0:
-      idx = (frame//10) % 4
-      can_sends.extend(hondacan.create_ui_commands(self.packer, pcm_speed, hud, CS.CP.carFingerprint, CS.is_metric, idx, CS.CP.openpilotLongitudinalControl, CS.stock_hud))
 
+    accel = actuators.gas - actuators.brake
+
+    # TODO: pass in LoC.long_control_state and use that to decide starting/stoppping
+    stopping = accel < 0 and CS.out.vEgo < 0.3
+    starting = accel > 0 and CS.out.vEgo < 0.3
+
+    # Prevent rolling backwards
+    accel = -1.0 if stopping else accel
+    apply_accel = interp(accel, P.BOSCH_ACCEL_LOOKUP_BP, P.BOSCH_ACCEL_LOOKUP_V)
+
+    gas_mult = interp(CS.out.vEgo, [0.0, 1.0], [2.0, 1.0])
+    pcm_speed = max(0.0, CS.out.vEgo + gas_mult * apply_accel)
+
+    # This is needed otherwise accel to decel is not smooth
+    pcm_accel = int(clip(apply_accel, 0.0, 1.0) * 0xc6)
```

Comment:
nidec is using bosch accel breakpoints/values?

Reply:
Yeah lemme rename